src/sentry/event_manager.py

```diff
@@ -2356,10 +2356,14 @@ def _calculate_event_grouping(
     Main entrypoint for modifying/enhancing and grouping an event, writes
     hashes back into event payload.
     """
+    load_stacktrace_from_cache = event.org_can_load_stacktrace_from_cache
```

Comment:
This property is used deeper in the stack of this code execution (line 2370 will help us reach [the following code](https://github.com/getsentry/sentry/blob/61c4e334313a4cc564bc8445820101c23a0247ff/src/sentry/eventstore/models.py#L415-L426)).